src/createDataDictionary.m

```diff
@@ -2,29 +2,18 @@
 
 function createDataDictionary(cfg, logFile)
     %
-    % Short description of what the function does goes here.
+    % It creates the data dictionary to be associated with a _events.tsv file. It will create empty
```

Comment:
Also the function can use the content of `cfg` to create those data dictionary so it will only create empty fields if the user has not set them before.